politeiawww/cmd/piwww/editproposal.go

```diff
@@ -70,6 +71,17 @@ func (cmd *EditProposalCmd) Execute(args []string) error {
 		}
 
 		md = b.Bytes()
+
+		// Check if cachedpropsoal got a linkto
+		// Get proposal
+		pdr, err := client.ProposalDetails(cmd.Args.Token,
+			&v1.ProposalsDetails{})
+		if err != nil {
+			return err
+		}
+		if pdr.Proposal.LinkTo != "" {
```

Comment:
You're missing metadata.  See the v1.go `ProposalMetadata` struct.  Also, move this down to where the other metadata is being handled so all the metadata code is in one place.